packages/vue-client/src/components/boards/MulticolorGraphBoard.vue

```diff
@@ -0,0 +1,129 @@
+<script lang="ts">
```

Comment:
Awesome we're building this on a Multicolor board - I've been considering converging on the multicolor board for grid baduk too (#137).

Reply:
Yeah, I think it would be cool if we could reduce the number of different board implementations 😆